src/query.ts

```diff
@@ -63,8 +63,21 @@ async function run(): Promise<void> {
         { continueOnError: false, retentionDays: 1 }
       );
     }
-  } catch (error: any) {
-    setFailed(error.message);
```

Comment:
Do you still want to call `setFailed` (probably at the end of this `catch` block) so that the job is marked as failed? Otherwise will it be marked as a success? If a user follows the logs link we want them to be able to easily tell which jobs failed.

Reply:
Yes, that's a good point! I will change it back.

(`setFailed` means that the entire action is marked as "failed" too, even if most of the repos succeeded. But I think that's still clearer than having the failed jobs marked as "success".)

Reply:
Could setting `continue-on-error: true` here help? Would need to test it but I think that would mean we could use `setFailed` to mark this step as failed but without the whole workflow being marked as a failure.